charts/renku/values.yaml

```diff
@@ -588,6 +588,17 @@ tests:
   image:
     repository: renku/tests
     tag: '0.2.0-rc4'
+  users_json:
+    - username: bwayne
+      firstName: Bruce
+      lastName: Wayne
+      email: bwayne@example.com
+      password: IamBatman
```

Comment:
Leave the passwords out of the defaults

Reply:
what to do then? it could be overly complicated if you want to generate one and insert it using the helm template functions

Reply:
All you need to do is something like:

```
test-users:
  - user1:
     username: bwayne
     firstName: Bruce
     lastName: Wayne
     email: bwayne@example.com
  - user2:
     username: ckent
     firstName: Clark
     lastName: Kent
     email: ckent@example.com
```

And in the template just put in the password.

Reply:
I don't think I follow. Did you check the template? It's using `toJson` so it's not possible

Reply:
right... the question then is why do we need the test users in `values.yaml`? They can just be in `secrets.yaml` with the JSON written out and the password field templated.

Reply:
I still don't see what is the problem with the current situation. We can comment the example values.

Reply:
Ok -- for the time being we can add a comment to `values.yaml` -- one easy way out is to change the `users.json` to yaml in `renku-demo` which would make it easier to template.